tests/api/id/[form]/retrieval.test.js

```diff
@@ -508,4 +508,67 @@ describe("/api/retrieval", () => {
       expect(logMessage.error.mock.calls[0][0]).toEqual(new Error("some error"));
     });
   });
+
+  describe("GET", () => {
+    it("Should return a list of form responses including Security attributes", async () => {
+      const token = jwt.sign(
+        {
+          email: "test@cds-snc.ca",
+          form: 100,
+        },
+        process.env.TOKEN_SECRET,
+        {
+          expiresIn: "1y",
+        }
+      );
+      const { req, res } = createMocks({
+        method: "GET",
+        headers: {
+          "Content-Type": "application/json",
+          Origin: "http://localhost:3000/api/retrieval?numRecords=4",
+          authorization: `Bearer ${token}`,
+        },
+        query: {
+          form: "100",
+        },
+      });
+      executeQuery.mockImplementation((client, sql) => {
+        if (
+          sql.includes(
+            "SELECT 1 FROM form_users WHERE template_id = ($1) and email = ($2) and temporary_token = ($3) and active = true"
+          )
+        ) {
+          return {
+            rows: [{ column: 1 }],
+            rowCount: 1,
+          };
+        }
+      });
+      const dynamodbExpectedReponses = {
+        Items: [
+          { FormID: "01", SubmissionID: "51", SecurityAttribute: "Protectd B" },
```

Comment:
Are there FormIDs that actually have leading zeros?  ie: `FormID : "01"`.

Reply:
There aren't yet. In the future we may completely drop a numerical order so i don't think it makes any difference in this scenario.

Reply:
There aren't yet. In the future we may completely drop a numerical order so i don't think it makes any difference in this scenario.